samples/generated_samples/snippet_metadata_google.pubsub.v1.json

```diff
@@ -8,7 +8,7 @@
     ],
     "language": "PYTHON",
     "name": "google-cloud-pubsub",
-    "version": "2.15.2"
+    "version": "0.1.0"
```

Comment:
I believe this version reset shouldn't matter, even though it's confusing. Some confirm?

Reply:
Yes, it doesn't matter, it will be reset by release-please (example: https://github.com/googleapis/python-pubsub/commit/d8e8a31bf4f4e1866e4cc57985e15f626e6b59e9)